### PR TITLE
Update info for Simina Boca

### PR DESCRIPTION
**Pull request by SiminaB**
I put the sentence "Work initially conducted at Georgetown University Medical Center with writing, reviewing, and editing continued while working at AstraZeneca" under COI, though it may work better somewhere under Contributions (not sure if it would fit there though.)

### Checklist

- [ X] Text is formatted so that each sentence is on its own line.

- [ X] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Review by agitter — APPROVED** [review]
Putting it here makes the most sense to me.

**Review by rando2 — APPROVED** [review]

File: content/metadata.yaml [hunk 1]
```diff
@@ -190,8 +190,10 @@ authors:
       confirmed: Yes
     affiliations:
       - Innovation Center for Biomedical Informatics, Georgetown University Medical Center, Washington, District of Columbia, United States of America
+      - Early Biometrics & Statistical Innovation, Data Science & Artificial Intelligence, R & D, AstraZeneca, Gaithersburg, Maryland, United States of America
     coi:
-      string: Currently an employee and minor share holder at AstraZeneca, Gaithersburg, MD, USA.
+      string: Currently an employee at AstraZeneca, Gaithersburg, MD, USA, may own stock or stock options. 
+      Work initially conducted at Georgetown University Medical Center with writing, reviewing, and editing continued while working at AstraZeneca.
```
Comment (rando2) [inline content/metadata.yaml:196]:
```suggestion
      string: Currently an employee at AstraZeneca, Gaithersburg, MD, USA, may own stock or stock options. Work initially conducted at Georgetown University Medical Center with writing, reviewing, and editing continued while working at AstraZeneca.
```

**Review by rando2 — APPROVED** [review]

**Comment by rando2** [comment]
This looks good to me! I think the build broke because of the line break, so trying it without

**Comment by agitter** [comment]
> I think the build broke because of the line break, so trying it without

Good catch.  That should fix the build error.

**Pull request merged**